### DISCUSSION TITLE
How do I use European numbering style for theorems/definitions etc.?

### DISCUSSION THREAD

**Discussion by olynch** [quarto-dev/quarto-cli#7686] [Q&A]
### Description

By European numbering style, I mean that all environments share a counter, so instead of Theorem 1.1, Definition 1.1, Proposition 1.1, it would be Theorem 1.1, Definition 1.2, Proposition 1.3.

**Comment by cderv** [comment]
@cscheid can we share counter across type ?  🤔

**Reply by cscheid** [reply]
Unfortunately no, not that I'm aware of.

This would be a pretty large change to the implementation. For example, we'd need cooperation from LaTeX and Typst's systems. I'm reluctant to add this to #4944 this late as well.

**Reply by olynch** [reply]
LaTeX definitely supports this; amsthm allows putting many different theorem types on a single counter. I don't know about typst though.
